src/juju/jimm-facade.ts

```diff
@@ -3,10 +3,24 @@ import { autoBind } from "@canonical/jujulib/dist/api/utils";
 
 import type { Controller } from "store/juju/types";
 
+// As typed in JIMM:
+// https://github.com/canonical/jimm/blob/1da76ed2d8dba741f5880f32c073f85f7d518904/api/params/params.go#L344
```

Comment:
This link and the one on line 14 can only be accessed internally. Is this ok, considering the open source nature of this project and that users can't access it externally?